packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -66,19 +68,26 @@ class RouteConfig {
     RouteConfig? parent,
   ) {
     assert(!reader.isNull, 'reader should not be null');
-    final ConstantReader pathValue = reader.read('path');
-    if (pathValue.isNull) {
-      throw InvalidGenerationSourceError(
-        'Missing `path` value on annotation.',
-        element: element,
-      );
-    }
+    final InterfaceType type = reader.objectValue.type! as InterfaceType;
+    // Ignore the deprected `element2` so that the "downgraded_analyze" CI step
+    // passes.
+    //ignore: deprecated_member_use
+    final bool isShellRoute = type.element2.name == 'TypedShellRoute';
```

Comment:
this should use `element`

Reply:
Changed to element in [4600f37](https://github.com/flutter/packages/pull/3439/commits/4600f3795b58653eff4618e96fdf0c8fbafcdb31)

Reply:
Actually I think that was intended. If I use element, the `downgraded_analyze` step of the CI fails (that's the comment above). How shall I handle this?

Reply:
we have been using the element in route_config.dart. I think you need to add a space inbetween ignore and slash?
```
// ignore: deprecated_member_use
```

Reply:
Ok I've added the same comment as in the other occurences of `element` in the code [9f8b067](https://github.com/flutter/packages/pull/3439/commits/9f8b0670b911abe83ad8bc00a13ae1ebab0bea2b)